device/mellanox/x86_64-mlnx_msn2700-r0/pmon_daemon_control.json

```diff
@@ -1,7 +1,7 @@
 {
     "skip_ledd": true,
     "skip_fancontrol": true,
-    "delay_xcvrd": true,
+    "delay_xcvrd": false,
```

Comment:
do we need to specify this flag explicitly as 'false' or we can just remove it?

Reply:
I'm not sure about removing this, because for some systems which are not going to participate in the fastboot it could be useful, but on the other hand leaving the `delay_xcvrd: true` going to increase an interface link up time up to 3 min